src/components/Card/Card.js

```diff
@@ -0,0 +1,21 @@
+import {inspirations} from '../../data';
+import ArtistCard from '../ArtistCard/ArtistCard';
+
+import {InspirationCard} from './Card.styled';
+export default function Card() {
+	console.log(inspirations);
```

Comment:
Please delete unnecessary console.log's

Reply:
thank you

Reply:
@stefankorder could you checkt it again please? I've solved the merge conflict and deleted the console.log my git hub actions are active now